### PR TITLE
Clean helper messages for configuration

### PR DESCRIPTION
**Pull request by qsantos**
Fixed a few types, removed redundancies and clarified some explanations.

I tried to make the messages more consistent. I may have mis-interpreted some.

### CODE REVIEW

**Comment by radare** [comment]
Thanks! looks way better now

**Comment by qsantos** [comment]
Thanks. By the way, why do almost all pull request get the status "closed" instead of "merged"? Is it due to the hight number of commits on the repository making it harder to merge?

File: libr/core/config.c [hunk 1]
```diff
@@ -1001,8 +1001,8 @@ R_API int r_core_config_init(RCore *core) {
 	SETI("asm.maxrefs", 5,  "Maximum number of xrefs to be displayed as list (use columns above)");
 	SETPREF("asm.bytes", "true",  "Display the bytes of each instruction");
 	SETPREF("asm.flagsinbytes", "false",  "Display flags inside the bytes space");
-	SETPREF("asm.midflags", "true", "Realign disassembly if there's a flag in the middle of an instruction");
-	SETPREF("asm.cmtflgrefs", "true", "Show comment flags associated to branch referece");
+	SETPREF("asm.midflags", "true", "Realign disassembly if there is a flag in the middle of an instruction");
+	SETPREF("asm.cmtflgrefs", "true", "Show comment flags associated to branch refenrece");
```
Comment (alvarofe) [inline libr/core/config.c]: is reference, isn't?

Comment (qsantos) [inline libr/core/config.c]: Nice catch, I should have proof-read my proof-reading.

**Comment by alvarofe** [comment]
I catched it by casuality jeje

**Pull request closed without merging**